tools-v2/pkg/cli/command/curvefs/warmup/add/add.go

```diff
@@ -47,7 +47,7 @@ $ curve fs warmup add /mnt/warmup # warmup all files in /mnt/warmup`
 )
 
 const (
-	CURVEFS_WARMUP_OP_XATTR      = "curvefs.warmup.op"
+	CURVEFS_WARMUP_OP_XATTR      = "curvefs.warmup.add"
```

Comment:
Is it okay to modify this way?

Reply:
this will be taken care of within the context of https://github.com/opencurve/curve/pull/2652/files#diff-a5860c1b36cdc66dd46550e8c558eb04e210e191e9550327dcf0599e5f1dc0a0L806